platform-api/che-core-api-git/src/main/java/org/eclipse/che/api/git/GitProjectType.java

```diff
@@ -22,11 +22,17 @@
 public class GitProjectType extends TransientMixin {
 
     public static final String VCS_PROVIDER_NAME = "vcs.provider.name";
+    public static final String CURRENT_BRANCH_NAME = "current.branch.name";
+    public static final String REPOSITORY_REMOTES = "repository.remotes";
```

Comment:
Actually this variant is also [possible](https://github.com/codenvy/codenvy-dev-resources/wiki/Formatting#fields-alignment), but agree alligment is preferable.